mlflow/tracking/_model_registry/utils.py

```diff
@@ -12,11 +12,11 @@
 
 # NOTE: in contrast to tracking, we do not support the following ways to specify
 # the model registry URI:
-#  - via a utility method like tracking.tracking.utils.set_tracking_uri
 #  - via environment variables like MLFLOW_TRACKING_URI, MLFLOW_TRACKING_USERNAME, ...
```

Comment:
is this comment about `MLFLOW_TRACKING_URI` still accurate?

Reply:
yeah. maybe it would be less confusing if it said `MLFLOW_REGISTRY_URI` ?